setup.py

```diff
@@ -100,4 +100,5 @@ def get_version():
     project_urls={
         "Documentation": "http://docs.getmoto.org/en/latest/",
     },
+    data_files=[('', ['moto/dynamodb2/parsing/reserved_keywords.txt'])],
```

Comment:
This incorrectly causes this file to be installed into the root of the Python package installation directory.

Reply:
Can you check whether https://github.com/spulec/moto/pull/2925 works for your environment?